src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java

```diff
@@ -130,7 +130,7 @@ public boolean cloneWorld(String oldName, String newName) {
         }
 
         // Check for valid world name
-        if (!(isValidWorldName(oldName) && isValidWorldName(newName))) {
+        if (!(WorldNameChecker.isValidWorldName(oldName) && WorldNameChecker.isValidWorldName(oldName))) {
```

Comment:
Shouldn't this be:
```diff
-if (!(WorldNameChecker.isValidWorldName(oldName) && WorldNameChecker.isValidWorldName(oldName))) {
+if (!(WorldNameChecker.isValidWorldName(oldName) && WorldNameChecker.isValidWorldName(newName))) {
```

Reply:
Probably a copy paste errors, can open a PR to fix it